### ISSUE TITLE
Application not updated on redeploy

### ISSUE THREAD

**Issue by Kolhor** [zodern/meteor-up#489]
Hi. I deployed a project on a DigitalOcean server using mup which works totally fine. It had a few bugs I wanted to fix so I did so and made sure they worked on localhost before redeploying.

Tried mup setup, deploy with only SUCCESS as answers on everything, but server is still running old version of my app...
Even tried mup reconfig, but it didn't help...

Tried rebooting server with sudo reboot -r now. When it came back online it still ran old version... No error codes in mup logs or anything...

When I deploy I only get SUCCESS criteria back... I have only found one other post with similar issure on stackoverflow...
[http://stackoverflow.com/questions/30096504/mup-deployed-site-not-displaying-after-update](url)

Apparently he gave up on DigitalOcean and went to Galaxy instead...

Not sure what more to do... First time I deploy anything so I'm all new at this.
help would be greatly appreciated..

EDIT: Well, what do you know... mup worked just the way it was supposed to: Perfectly. The fault was all with me...

**Comment by zodern** [comment]
@Kolhor could you please share what the problem was?

**Comment by Kolhor** [comment]
The problem was right in front of the keyboard. 

First I deployed without SSL (for some reason). This meant that the verification mail new users got which started with "https://" would not work.

When I told my co-developer this (who had implemented the email system) he quickly made a "hot fix" which would solve the problem.

the url which is used as a parameter in the verification-email-step would have a line of code added;
url.replace("https", "http");
Didn't really think of it... Had a replace in it! Looked good to me.

Now this didn't really do anything...

Spent hours looking at mup and googling mup problems when the entire problem was the code we'd written.
Later on I changed the line of code to;
var res = url.replace("https", "http");

Later on, I added SSL (because what respectable website doesn't have SSL?)

Actually ended up changing that bit of code once more... (not that it really matters).
It referred to "https://IP-ADDRESS/VERIFICATION-TOKEN".
Had to use "var res = url.replace("IP-ADDRESS", "DOMAIN NAME"), so that the web-browser wouldn't give a "not secure site" warning.

Well... anyway, that's my rookie mistake. Bad code. Pretty embarrassing and sorry for creating issue where there was none.
